lelkek.FrogBox/web.css

```diff
@@ -0,0 +1,238 @@
+*/
+
+body {
+  margin: 0
+}
+
+a {
+  text-decoration: none;
+  color: #000
+}
+
+hr {
+  display: block;
+  height: 1px;
+  border: 0;
+  border-top: 1px solid #ccc;
+  margin: 1em 0;
+  padding: 0;
+  width: 60vw;
+  margin: auto;
+}
+
+.roundedimg {
+  border-radius: 50%;
+}
+
+.arrowhover:hover .arrow_box {
+  visibility: visible;
+  opacity:1;
+  transition:opacity 0.15s linear;
+}
+.arrowhover:hover .nametext {
+  display:none;
+}
+
+#logo {
+  height: 80px;
+  width: 80px
+}
+
+.nametext {
+  margin: 10px 0px;
+}
+.arrow_box {
+  visibility: hidden;
+  position: relative;
+  background: #ffffff;
+  -webkit-box-shadow: 0px 0px 29px -5px rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 0px 0px 29px -5px rgba(0, 0, 0, 0.75);
+  box-shadow: 0px 0px 29px -5px rgba(0, 0, 0, 0.75);
+  z-index: 100;
+  padding: 20px;
+  opacity: 0;
+}
+
+.arrow_box:after {
+  bottom: 100%;
+  left: 50%;
+  border: solid transparent;
+  content: " ";
+  height: 0;
+  width: 0;
+  position: absolute;
+  pointer-events: none;
+  border-color: rgba(255, 255, 255, 0);
+  border-bottom-color: #ffffff;
+  border-width: 30px;
+  margin-left: -30px;
+}
+
+.navbarmp {
+  -webkit-box-shadow: 0px -9px 66px 0px rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 0px -9px 66px 0px rgba(0, 0, 0, 0.75);
+  box-shadow: 0px -9px 66px 0px rgba(0, 0, 0, 0.75);
+  display: flex;
+  flex-wrap: nowrap;
+  overflow: hidden;
+  align-items: center;
+  flex-direction: row;
+  height: 100px;
+  position: relative;
+  z-index: 10;
+  padding: 0 15vw
+}
+
+.bold {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 700;
+}
+
+.regular {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 300;
+}
+
+.navbar {
+  /* -webkit-box-shadow: 0px -9px 66px 0px rgba(0,0,0,0.40);
+-moz-box-shadow: 0px -9px 66px 0px rgba(0,0,0,0.40);
+box-shadow: 0px -9px 66px 0px rgba(0,0,0,0.40); */
+  display: flex;
+  flex-wrap: nowrap;
+  overflow: hidden;
+  align-items: center;
+  flex-direction: row;
+  height: 100px;
+  position: relative;
+  z-index: 10;
+  padding: 0 15vw;
+}
+
+.atcbutton {
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  justify-content: center;
+  float: right;
+  margin-right: 3px;
+}
+
+.divided {
+  text-decoration: none;
+  font-family: 'Montserrat', sans-serif;
+  display: inline-block;
+  position: relative;
+  width: 100px;
+  text-align: center;
+  align-content: center;
+  padding: 20px 0px;
+  flex-wrap: nowrap;
+  transition: color 0.2s
+}
+
+.divided:hover {
+  color: #b5d849;
+  transition: color 0.2s;
+}
+
+.fpimage {
+  width: 100vw;
+  height: 60vh;
+  background-image: url("http://i.imgur.com/ZzGM2SO.jpg");
```

Comment:
I can see the argument for not wanting binary files in a repo, but this makes development on the website require internet access, something not guaranteed at your own school.